examples/anthos-service-mesh-multicluster/mysql-docker/setup.sh

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Docker environment variables
+export MYSQL_DOCKER_REPO="mysql-docker-repo"
```

Comment:
Seems for example we should default to Google Artifact Registry in the project in question https://cloud.google.com/artifact-registry/docs

Reply:
Yeah, I had done this but forgot to update this part of the variable assignment. Fixed